sys/kern/devfs.c

```diff
@@ -89,7 +130,13 @@ int devfs_makedev(devfs_node_t *parent, const char *name, vnodeops_t *vops,
   if (error)
     return error;
 
-  dn->dn_vnode = vnode_new(V_DEV, vops, data);
+  devfs_add_default_vops(vops);
+  vnodeops_init(vops);
+
+  dn->dn_vnode = vnode_new(V_DEV, vops, dn);
+  dn->dn_data = data;
+  dn->dn_mode =
+    S_IFCHR | S_IRUSR | S_IWUSR | S_IRGRP | S_IWGRP | S_IROTH | S_IWOTH;
```

Comment:
`S_IWOTH` – others have write permission. Is it ok?

Reply:
We have decided that it is ok for now.